dashboard/test/integration/lessons_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class LessonsTest < ActionDispatch::IntegrationTest
+  self.use_transactional_test_case = true
```

Comment:
what does this do?

Reply:
it literally

https://github.com/code-dot-org/code-dot-org/blob/23b9c75e79f6c9b94f3c6f9894030e7a57daacb5/dashboard/test/testing/transactional_test_case.rb#L7

and then rolls it back after. the result is that changes made to the database are visible within the current test case, but never leak over into other test cases.

https://www.tutorialspoint.com/mysql/mysql-transactions.htm

Reply:
Thank you!

Reply:
I thought this was on by default now? is it different for `IntegrationTest`? https://blog.bigbinary.com/2016/05/26/rails-5-renamed-transactional-fixtures-to-transactional-tests.html

Reply:
TIL! I've removed this statement. A quick experiment confirms what you and the blog are saying.